### ISSUE TITLE
[Bug] [Shell] TitleView glitches with ToolbarItems in Top tab bar

### ISSUE THREAD

**Issue by NunoBem** [xamarin/Xamarin.Forms#7244]
### Description
Having a Top tab bar defined in Shell where inside each one will have a Shell.TitleView and ContentPage.ToolbarItems causes glitches between them.

[Screen recording](https://bit.ly/2Z9QG3T)

When opening a first page the TitleView and ToolbarItem are fine, changing to a different tab will put a duplicate part of the TitleView in front of the ToolbarItem and stays like that for every tab.

### Steps to Reproduce

1. Create a new Shell project and define a Top tab bar layout with more than 1 page.
2. In each xaml page, define the TitleView and ToolbarItem:

```
<Shell.TitleView>
     <StackLayout>
         Text="Title" />
     </StackLayout>
</Shell.TitleView>

<ContentPage.ToolbarItems>
     <ToolbarItem
         IconImageSource="ic_edit"
         Order="Primary"
         Priority="0" />
</ContentPage.ToolbarItems>
```

### Expected Behavior
No glitch and overlap of text above ToolbarItem.

### Actual Behavior
Duplicated TitleView above the ToolbarItem.

### Basic Information

- Version with issue:
- Last known good version: Xamarin.Forms 4.2.0.709249
- IDE: Visual Studio 16.2.3
- Affected Devices: Only on Android

### Screenshots
[Screen recording](bit.ly/2Z9QG3T)

![Screenshot_20190822-133442](https://user-images.githubusercontent.com/524734/63519345-5210da80-c4e2-11e9-9830-8db99ec9fb6c.jpg)

![Screenshot_20190822-133503__01](https://user-images.githubusercontent.com/524734/63519362-59d07f00-c4e2-11e9-8072-442be73fbfc9.jpg)

![Screenshot_20190822-133508__01](https://user-images.githubusercontent.com/524734/63519375-5e953300-c4e2-11e9-9263-59b7c2e5cb5a.jpg)

**Comment by MSiccDev** [comment]
I am having the same problem. It does not happen with using the Title Property of ContentPage.

**Comment by NunoBem** [comment]
Tested in iOS to check, and is an Android only issue.

**Comment by samhouts** [comment]
closed by #7276
